VCFoodDeliveryApp/Models/Cart.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace VCFoodDeliveryApp.Models
+{
+    public class Cart
+    {
+        [StringLength(450, ErrorMessage = "Too many users created")]
+        public string Id { get; set; }
+
+
+        public string ApplicationUserId { get; set; }
```

Comment:
This seems to have generated the FK you want in the migration, but it's usually a good idea to pair FKs with a navigation property, too. Use `[ForeignKey]` if the names of the properties don't align with the standard `NavPropId` convention.